cmd/rook/userfacing/client/client.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Do we need the `userfacing` package? Or could it just be `cmd/rook/client/client.go`?

Reply:
I'll just use the `rook.NewContext()` command.